microsoft-edge/devtools-guide-chromium/sources/index.md

```diff
@@ -87,49 +279,292 @@ script.integrity = 'sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=';
+
+### Edit a CSS file
+
+There are two ways to edit CSS in DevTools:
 
-## Debug JavaScript  
+*   In the **Elements** tool, [Edit CSS font styles and settings in the Styles pane][DevToolsInspectStylesEditFonts] (recommended in most cases).
+*   In the **Sources** tool, in the **Editor** pane, edit the CSS file as a file listing that directly shows the file.
 
-Rather than using `console.log()` to infer where your JavaScript is going wrong, consider using the Microsoft Edge DevTools debugging tools, instead.  The general idea is to set a breakpoint, which is an intentional stopping place in your code, and then step through the runtime of your code, one line at a time.  As you step through the code, you may display and change the values of all currently-defined properties and variables, run JavaScript in the **Console**, and more.
+The Sources tool supports directly editing a CSS file.  For example, if you edit the CSS file from the tutorial [Edit files with Workspaces][DevtoolsGuideChromiumWorkspacesIndex] to match the style rule below, the `H1` element in the upper left of the rendered webpage changes to green:
 
-Navigate to [Get Started With Debugging JavaScript][DevtoolsGuideChromiumJavascriptIndex] to learn the basics of debugging in DevTools.
+```css
+h1 {
+  color: green;
+}  
+```
 
-:::image type="complex" source="../media/debugging.msft.png" alt-text="Debug JavaScript" lightbox="../media/debugging.msft.png":::
-   Debug JavaScript  
+:::image type="complex" source="../media/edit-css.msft.png" alt-text="Edit CSS in the Editor pane to change the text color of the H1 heading to green" lightbox="../media/edit-css.msft.png":::
+   Edit CSS in the **Editor** pane to change the text color of the `H1` heading to green  
 :::image-end:::  
 
-## Set up a Workspace  
+CSS changes take effect immediately; in most cases, no saving is needed.
+
+**Related articles:**
+
+*   [Edit CSS font styles and settings in the Styles pane][DevToolsInspectStylesEditFonts]
+
+*   [DevTools for beginners: Get started with CSS][DevToolsBeginnersCss] - tutorial
+
+### Opening the DevTools code editor when using a tool other than the Sources tool
+
+The main place to use the code editor of DevTools is within the **Sources** tool.  But sometimes you need to access other tools, such as the **Elements** tool or the **Console** panel, while making changes to your site.  There is a way to have the DevTools code editor open alongside other panels.  
+
+1.  Choose a tool other than the **Sources** tool, such as the **Elements** tool.  
+1.  Select `Control`+`Shift`+`P` \(Windows, Linux\) or `Command`+`Shift`+`P` \(macOS\).  The **Command Menu** opens.  
+1.  Type `QS`, then choose **Show Quick Source**.  At the bottom of the DevTools window, the **Quick Source** panel appears, displaying the last file you edited in the **Sources** tool, within the code editor from the **Sources** tool.  
```

Comment:
I think for consistency, I'd prefer if our instructions were "Type `Quick Source`" and then have them select and run the command from there. It's a little more robust this way if we ever add a command in the future that also has a Q and an S in it